src/D2L.Bmx/WindowsParentProcess.cs

```diff
@@ -0,0 +1,46 @@
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace D2L.Bmx;
+
+internal partial class WindowsParentProcess {
+
+	[DllImport( "ntdll.dll", EntryPoint = "NtQueryInformationProcess" )]
+	internal static extern int NtQueryInformationProcess(
+		IntPtr processHandle,
+		int processInformationClass,
+		ref PROCESS_BASIC_INFORMATION processInformation,
+		int processInformationLength,
+		out int returnLength
+	);
```

Comment:
This is part of internal windows api and the windows docs recommends something from user level of the api

Reply:
And I think that's only because they can change the structure or output in a future update and break the current setup